Assets/MixedRealityToolkit/Utilities/Gltf/Schema/GltfMaterial.cs

```diff
@@ -0,0 +1,91 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Utilities.Gltf.Schema
+{
+    /// <summary>
+    /// The material appearance of a primitive.
+    /// https://github.com/KhronosGroup/glTF/tree/master/specification/2.0/schema
+    /// </summary>
+    [Serializable]
+    public class GltfMaterial : GltfChildOfRootProperty
+    {
+        /// <summary>
+        /// A set of parameter values that are used to define the metallic-roughness
+        /// material model from Physically-Based Rendering (PBR) methodology.
+        /// </summary>
+        public GltfPbrMetallicRoughness pbrMetallicRoughness;
```

Comment:
Per coding conventions, public fields should be capitalized.

Reply:
Synced offline: onboard with this naming based on khronos schema declaration